build.gradle

```diff
@@ -539,30 +560,123 @@ publishing {
                     developer {
                         id = 'cschaebe'
                         name = 'Christoph Schaebel'
-                        email = 'christoph.schaebel@dc-square.de'
-                        organization = 'dc-square GmbH'
-                        organizationUrl = 'https://www.dc-square.de'
+                        email = 'christoph.schaebel@hivemq.com'
+                        organization = 'HiveMQ GmbH'
+                        organizationUrl = 'https://www.hivemq.com'
+                    }
+                    developer {
+                        id = 'lbrandl'
+                        name = 'Lukas Brandl'
+                        email = 'lukas.brandl@hivemq.com'
+                        organization = 'HiveMQ GmbH'
+                        organizationUrl = 'https://www.hivemq.com'
+                    }
+                    developer {
+                        id = 'flimpoeck'
+                        name = 'Florian Limpoeck'
+                        email = 'florian.limpoeck@dc-square.de'
+                        organization = 'HiveMQ GmbH'
+                        organizationUrl = 'https://www.hivemq.com'
+                    }
+                    developer {
+                        id = 'sauroter'
+                        name = 'Georg Held'
+                        email = 'georg.held@hivemq.com'
+                        organization = 'HiveMQ GmbH'
+                        organizationUrl = 'https://www.hivemq.com'
+                    }
+
+                    developer {
+                        id = 'SgtSilvio'
+                        name = 'Silvio Giebl'
+                        email = 'silvio.giebl@hivemq.com'
+                        organization = 'HiveMQ GmbH'
+                        organizationUrl = 'https://www.hivemq.com'
+                    }
+                }
+                scm {
+                    connection = 'scm:git:git@github.com:hivemq/hivemq-community-edition.git'
+                    developerConnection = 'scm:git:git@github.com:hivemq/hivemq-community-edition.git'
+                    url = 'https://github.com/hivemq/hivemq-community-edition/tree/master'
+                }
+            }
+        }
+
+        embedded(MavenPublication) {
+            from components.java
+            artifacts = [embeddedJar]
+            artifactId 'hivemq-community-edition-embedded'
```

Comment:
artifact sourcesJar is missing (not needed if we use withSourcesJar as mentioned above)

Reply:
also we need a javadocJar, this is a requirement for maven central

Reply:
👍